go-controller/pkg/ovn/multipolicy_test.go

```diff
@@ -540,9 +541,10 @@ var _ = ginkgo.Describe("OVN MultiNetworkPolicy Operations", func() {
 		})
 
 		ginkgo.DescribeTable("correctly adds and deletes pod IPs from secondary network namespace address set",
-			func(topology string, remote bool) {
+			func(topology string, remote bool, expectMissingMappingFailure bool) {
```

Comment:
oh boy ... could we maybe turn this into a builder ?... If you do, instead of adding a boolean, pass directly the expected failure or something like that.